DomainEntity/CertificationExamResult.metaed

```diff
@@ -21,3 +21,7 @@ Domain Entity CertificationExamResult
     descriptor CertificationExamStatus
         documentation "The status of the Certification Exam attempt."
         is optional
+    domain entity EdFi.StudentAssessment
```

Comment:
This is from v1 of the Plan design concept, but we will keep it even though it is not a part of the v5 design. See discussion in DATASTD-2116 for details